01-JS-DOM-APIs/javascript.js

```diff
@@ -0,0 +1,117 @@
+//1-4
+function show(){
+                  let obj = document.getElementById("hidden");
+                  if(obj.style.display == "block"){
```

Comment:
when doing conditionals try to always evaluate with ``===`` as it is more strict